commitlint/abbreviations.ts

```diff
@@ -1,186 +1,186 @@
 export let abbr = {
     "1 second": "1sec",
     "1second": "1sec",
-    "absolute": "abs",
-    "address": "addr",
-    "algorithm": "algo",
-    "allocate": "alloc",
-    "alternate": "alt",
-    "alternative": "alt",
-    "and": "&",
-    "application": "app",
-    "argument": "arg",
-    "asynchronous": "async",
-    "attribute": "attrib",
-    "authenticate": "auth",
-    "authentication": "auth",
-    "average": "avg",
-    "background": "bg",
-    "binary": "bin",
-    "bitcoin": "BTC",
-    "block": "blk",
-    "boolean": "bool",
-    "buffer": "buf",
-    "button": "btn",
-    "calculate": "calc",
-    "callback": "cb",
-    "channel": "chan",
-    "character": "char",
-    "collection": "coll",
-    "column": "col",
-    "command": "cmd",
+    absolute: "abs",
```

Comment:
@realmarv woah, seriously? I can't believe prettier would do this by default, remove the quotes for single words and leave the quotes for words with spaces? my god, can you investigate how to disable this behaviour please, we want all of them with double quotes

Reply:
Done

Reply:
Nice.